<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="InputA">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input A</string>
    </property>
   </widget>
   <widget class="QLabel" name="InputB">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input B</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_A">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_B">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="andgateBttn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>AND Gate</string>
    </property>
   </widget>
   <widget class="QLabel" name="gatelabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="orgateBttn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OR Gate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="notgateBttn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>NOT Gate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nandgateBttn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>NAND Gate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buffergateBttn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>BUFFER Gate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="norgateBttn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>NOR Gate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xorgateBttn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>XOR Gate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xnorgateBttn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>XNOR Gate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Boolean Expression:</string>
    </property>
   </widget>
   <widget class="QLabel" name="boolexplabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
